exporter/bmchelixexporter/exporter.go

```diff
@@ -0,0 +1,84 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package bmchelixexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/bmchelixexporter"
+
+import (
+	"context"
+	"errors"
+	"os"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/exporter"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/zap"
+)
+
+// BmcHelixExporter is responsible for exporting metrics to BMC Helix
+type BmcHelixExporter struct {
+	config            *Config
+	logger            *zap.Logger
+	version           string
+	telemetrySettings component.TelemetrySettings
+	metricsProducer   *MetricsProducer
+	metricsClient     *MetricsClient
+}
+
+// newBmcHelixExporter instantiates a new BMC Helix Exporter
+func newBmcHelixExporter(config *Config, createSettings exporter.Settings) (*BmcHelixExporter, error) {
+	if config == nil {
+		return nil, errors.New("nil config")
+	}
+
+	return &BmcHelixExporter{
+		config:            config,
+		version:           createSettings.BuildInfo.Version,
+		logger:            createSettings.Logger,
+		telemetrySettings: createSettings.TelemetrySettings,
+	}, nil
+}
+
+// pushMetrics is invoked by the OpenTelemetry Collector to push metrics to BMC Helix
+func (be *BmcHelixExporter) pushMetrics(ctx context.Context, md pmetric.Metrics) error {
+	be.logger.Info("Building BMC Helix payload")
+	helixMetrics, err := be.metricsProducer.ProduceHelixPayload(md)
+	if err != nil {
+		be.logger.Error("Failed to build BMC Helix payload", zap.Error(err))
+		return err
+	}
+
+	be.logger.Info("Sending BMC Helix payload")
```

Comment:
Is it common practice to log all this? I would have expected such messages ("Building", "Sending", etc.) to be in debug level, rather than Info. (but I'm not familiar with the logging in other components!)

Reply:
I think we can remove these two logs. I don't see what value they add, and after reviewing a few exporters, I didn't notice a similar practice to what I implemented here...

Reply:
This is going to be a noisey log and potentially provide little value, can you remove this?

Reply:
Yes, I will remove it.